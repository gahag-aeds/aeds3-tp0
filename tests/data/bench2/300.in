300 150
63 76 35 21 95 8 18 88 27 60 61 38 73 78 72 17 99 36 58 62 0 82 24 2 79 43 12 55 57 83 20 73 59 7 94 6 15 12 95 94 25 8 85 50 38 57 67 37 93 77 51 46 11 75 48 90 70 12 45 28 47 17 1 59 76 47 65 91 60 12 38 37 20 23 87 10 80 7 99 26 36 2 72 48 78 72 38 0 36 36 80 84 5 33 95 82 81 60 25 93 73 63 30 45 86 17 8 19 76 7 45 13 10 69 61 40 41 51 40 77 87 73 13 45 6 8 27 87 21 52 80 46 68 62 43 54 32 51 25 8 59 70 73 21 39 34 61 32 38 53 10 77 26 75 22 85 36 1 72 57 6 5 55 74 67 98 80 99 50 6 60 61 76 33 82 68 20 95 0 58 0 62 35 79 90 10 64 26 63 88 35 69 93 90 95 61 88 76 12 38 82 72 51 10 6 85 30 78 32 83 36 33 45 23 64 35 85 28 61 49 16 96 18 62 86 14 75 27 42 87 17 76 60 69 38 18 6 69 96 39 52 84 24 49 59 88 37 45 68 98 94 36 47 64 98 85 30 73 64 24 61 82 0 73 3 39 43 9 60 91 0 64 75 24 13 34 64 50 79 84 1 25 21 0 42 19 37 72 93 2 49 6 84 49 79 39 88 74 48 0 
Sub 66 165
Sum 150 249
Add 65 164
Sum 41 140
Sub 26 125
Max 79 178
Add 127 226
Min 191 290
Add 181 280
Max 159 258
Add 175 274
Sum 182 281
Sub 184 283
Max 148 247
Sub 34 133
Max 26 125
Sub 168 267
Min 72 171
Add 151 250
Sum 113 212
Sub 174 273
Sum 200 299
Add 19 118
Sum 110 209
Add 101 200
Sum 50 149
Add 149 248
Sum 142 241
Sub 24 123
Min 149 248
Sub 142 241
Min 89 188
Sub 50 149
Max 7 106
Add 81 180
Min 76 175
Add 37 136
Min 7 106
Sub 18 117
Max 169 268
Add 165 264
Sum 144 243
Add 151 250
Min 101 200
Sub 92 191
Sum 136 235
Sub 78 177
Max 59 158
Add 7 106
Min 180 279
Add 123 222
Min 132 231
Add 179 278
Min 141 240
Add 193 292
Min 32 131
Sub 190 289
Sum 93 192
Add 173 272
Max 13 112
Sub 42 141
Max 114 213
Add 53 152
Min 91 190
Sub 72 171
Sum 133 232
Sub 118 217
Max 60 159
Add 91 190
Sum 135 234
Add 29 128
Min 8 107
Add 31 130
Sum 131 230
Sub 108 207
Max 122 221
Sub 66 165
Sum 185 284
Sub 132 231
Max 73 172
Add 165 264
Max 54 153
Sub 156 255
Min 95 194
Sub 146 245
Min 149 248
Sub 78 177
Max 135 234
Add 41 140
Min 50 149
Add 99 198
Min 34 133
Sub 116 215
Sum 151 250
Add 77 176
Max 139 238
Add 35 134
Min 194 293
Add 199 298
Min 77 176
Add 81 180
Max 179 278
Sub 84 183
Sum 63 162
Add 109 208
Max 143 242
Sub 184 283
Sum 126 225
Sub 80 179
Max 67 166
Sub 150 249
Min 196 295
Sub 72 171
Max 106 205
Sub 42 141
Sum 17 116
Sub 6 105
Min 120 219
Add 103 202
Sum 198 297
Add 169 268
Max 1 100
Sub 184 283
Max 36 135
Add 103 202
Min 60 159
Add 175 274
Max 89 188
Add 5 104
Max 125 224
Add 117 216
Min 37 136
Add 103 202
Max 153 252
Add 99 198
Min 163 262
Add 95 194
Min 34 133
Sub 40 139
Sum 126 225
Sub 52 151
Sum 190 289
Add 121 220
Sum 123 222
Add 47 146
Max 163 262
Add 91 190
Min 52 151
Sub 34 133
Min 193 292
