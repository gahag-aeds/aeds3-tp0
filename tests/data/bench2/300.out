5270
4731
99
0
100
5578
100
97
-2
5192
5040
5066
4679
5455
3
2
99
-2
-1
98
5430
5
5448
95
0
5
5
-2
5009
100
103
2
5581
95
5560
-2
5579
101
5563
100
96
0
-1
100
-2
-2
5502
100
0
-1
103
4569
101
5281
99
0
97
4738
-3
5179
99
96
-4
101
101
-3
103
9
-3
5605
5548
5571
106
-4
-1
